tex/artikel/Evaluation.tex

```diff
@@ -67,11 +67,9 @@ \subsection{Wie ist der Stand heute?}
 Aktuell ist die Fachschaft in allen entscheidenden Institutionen für die Evaluation involviert und kann euer Feedback zur Evaluation direkt umsetzen.
 Ebenso achten wir auf die Einhaltung aller Evaluationsrichtlinien, damit die Studierenden direkt von Ihrem Feedback profitieren können.
 
-Nach wie vor werden die Umfragen von uns in Papierform durchgeführt.
-Um die Auswertung zu beschleunigen und Probleme zu umgehen, die sich bei einer analogen Evaluation nicht vermeiden lassen, wird die Evaluation nach und nach auf eine Online-Evaluation umgestellt. Diese erfolgen jedoch nach wie vor in Präsenz, also während der Vorlesung, um die Anzahl an Teilnehmern gegenüber einer analogen Evaluation konstant zu halten.
+Lange Zeit wurden die Umfragen in Papierform durchgeführt. Auch wenn es vorher schon erste Versuche gab, die Evaluation digital durchzuführen, mussten im letzten Jahr durch die Coronapandemie alle Umfragen Online durchgeführt werden. So dass wir je nach Umsetzung der Lehrveranstaltungen des nächsten Semesters auf Erfahrungen aus analoger und digitaler Evaluation zurückgreifen können. Generell gilt jedoch, dass die Evaluation während der Vorlesung durchgeführt wird, um die Anzahl an Teilnehmern gegenüber einer analogen Evaluation konstant zu halten.
```

Comment:
Hier steht 3 mal "durchführen" in zwei Sätzen. Ich würde vorschlagen das mittlere durch "abzuhalten" oder "auszuführen" etc. zu ersetzen.

Reply:
Ich schlage Hauke mal folgendes vor:
```suggestion
Lange Zeit wurden die Umfragen in Papierform durchgeführt. Auch wenn es vorher schon erste Versuche gab, die Evaluation digital umzusetzen, mussten im letzten Jahr durch die Coronapandemie alle Umfragen online stattfinden. So dass wir je nach Umsetzung der Lehrveranstaltungen des nächsten Semesters auf Erfahrungen aus analoger und digitaler Evaluation zurückgreifen können. Generell gilt jedoch, dass die Evaluation während der Vorlesung durchgeführt wird, um die Anzahl an Teilnehmern gegenüber einer analogen Evaluation konstant zu halten.
```